apps/internal-storybook/package.json

```diff
@@ -17,6 +17,7 @@
 		"@vitejs/plugin-react": "^4.7.0",
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
-		"storybook": "catalog:"
+		"storybook": "catalog:",
+		"vite": "^7.2.2"
```

Comment:
we should probably add vite to the catalog in `pnpm-workspace.yaml` and use it everywhere.